### PR TITLE
EventGrid: Fix multiple linter and swagger correctness/completeness issues for latest preview and stable API versions

### PR DESCRIPTION
**Pull request by ahamad-MS**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [X] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  02/05/2021

  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  02/05/2021

  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [X] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ahamad-MS** [comment]
Hello @qianwens .. can you please help with linter and breaking changes failures?

All our fixes are based on swagger correctness errors we received in s360...

**Comment by akning-ms** [comment]
@jianyexi for help on Lint or breaking change failure

**Comment by jianyexi** [comment]
@ahamad-MS  Currently , the default value for a model can only be a string,  it means that you can not give a default value for an object model. Below is the error position:
https://github.com/Azure/azure-rest-api-specs/blob/633673474aafc219fef0c6481bb6076427496072/specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json#L6373

**Comment by ahamad-MS** [comment]
Hello @jianyexi I addressed all the issues in swagger including the default values.

The checks seem to be stuck and not progressing. Can you please take a look and help sign off/merge this ASAP?

thanks

File: specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json [hunk 1]
```diff
@@ -6954,6 +6996,12 @@
           "$ref": "#/definitions/PartnerNamespaceProperties",
           "description": "Properties of the partner namespace.",
           "x-ms-client-flatten": true
+        },
+        "systemData": {
```
Comment (jianyexi) [inline specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json:7000]: Does this property really exist in the API response ?

Comment (ahamad-MS) [reply specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json:7000]: Yes it does..

**Review by Kishp01 — APPROVED** [review]

**Comment by ahamad-MS** [comment]
Hello @jianyexi @qianwens.. The comments are addressed. Please Let me know if additional info needed and what is expected ETA for sign off? I was asked for an ETA for this work

Thanks

**Review by jianyexi — APPROVED** [review]

**Comment by jianyexi** [comment]
@qiaozha  could you look at the js SDK failre ?

**Comment by mikhailshilkov** [comment]
I believe this PR has broken the specs: https://github.com/Azure/azure-rest-api-specs/issues/13167

**Pull request merged**